.github/workflows/integration-test.yml

```diff
@@ -981,11 +981,18 @@ jobs:
           timeout_minutes: 60 # EKS takes about 20 minutes to spin up a cluster and service on the cluster
           retry_wait_seconds: 5
           command: |
-            cd terraform/eks/daemon
+            if [ "${{ matrix.arrays.terraform_dir }}" != "" ]; then
```

Comment:
Should this logic apply to others as well for consistency?

Reply:
I think it should, but we should revisit this later.